src/applications/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -3,14 +3,16 @@ import Scroll from 'react-scroll';
 import _ from 'lodash';
```

Comment:
Out of scope, but I think we should probably pull in the specific merge function instead of the entire library here.

Reply:
I don't think we even need to use the `merge` function honestly. Planning on looping back on this one as well